std/algebra/emulated/sw_bw6761/hints.go

```diff
@@ -0,0 +1,110 @@
+package sw_bw6761
+
+import (
+	"math/big"
+
+	bw6761 "github.com/consensys/gnark-crypto/ecc/bw6-761"
+	"github.com/consensys/gnark/constraint/solver"
+	"github.com/consensys/gnark/std/math/emulated"
+)
+
+func init() {
+	solver.RegisterHint(GetHints()...)
+}
+
+// GetHints returns all hint functions used in the package.
+func GetHints() []solver.Hint {
+	return []solver.Hint{
+		pairingCheckHint,
+		finalExpHint,
+	}
+}
+
+func pairingCheckHint(nativeMod *big.Int, nativeInputs, nativeOutputs []*big.Int) error {
```

Comment:
Refactor with finalExpHint

Reply:
this one is different (different `mInv`), but I'll what I can do